### DISCUSSION TITLE
Roadrunner Compatibility with CARLA

### DISCUSSION THREAD

**Discussion by catac0mb** [carla-simulator/carla#7757] [Q&A] [answered]
Will roadrunner work with the latest version of CARLA, or should I build the 0.9.13 version of CARLA? I am trying to build the 0.9.13 version but the link to the 0.9.13 content files in ContentVersions.txt does not work.

**Comment by ADASDevelopment** [comment, marked as answer]
Hello. We are using RoadRunner R2023b and Carla latest version 0.9.15. There is no incompatibility. Carla just doesn't know bidirectional. I think it's something to do with Carla's side.
